spec/features/checkout_address_selection_spec.rb

```diff
@@ -24,6 +24,7 @@
 
     it "should only see shipping address form" do
       within("#shipping") do
+        uncheck 'order_use_billing'
```

Comment:
Are these `uncheck` lines required?  It might be a sign that we have something wrong in the use_billing checkbox logic.